src/figuresClasses.ts

```diff
@@ -1,11 +1,101 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: 'triangle' = 'triangle';
+
+  public color: 'red' | 'green' | 'blue';
+
+  public a: number;
+
+  public b: number;
+
+  public c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side lengths must be greater than zero');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error(
+        'Sides ' + a + ', ' + b + ' and ' + c + " can't form a triangle",
+      );
+    }
+
+    this.a = a;
+    this.b = b;
+    this.c = c;
+    this.color = color;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return parseFloat(area.toFixed(2));
+  }
+}
+
+export class Circle implements Figure {
+  public shape: 'circle' = 'circle';
+
+  public color: 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+  public radius: number;
 
-export class Circle implements Figure {}
+  constructor(color: 'red' | 'green' | 'blue', radius: number) {
+    if (radius <= 0) {
+      throw new Error('Radius must be greater than zero');
+    }
+
+    this.radius = radius;
+    this.color = color;
+  }
+
+  getArea(): number {
+    const area = Math.PI * this.radius * this.radius;
+
+    return Math.floor(area * 100) / 100;
```

Comment:
The `getArea` method for the `Circle` class uses `Math.floor` for rounding, which may not match the expected precision in the tests. Consider using `parseFloat(area.toFixed(2))` to ensure the area is rounded to two decimal places.